modules/ROOT/nav.adoc

```diff
@@ -80,24 +75,26 @@
  ** xref:add-another-output-transform-studio-task.adoc[Add Another Output Target]
  ** xref:move-transformations-separate-file-studio-task.adoc[Move Transformations to Separate Files]
 
-* xref:eu-cloud-configuration.adoc[Configure EU Cloud in Studio]
-
-* xref:pce-configuration.adoc[Configure Private Cloud Edition in Studio]
- ** xref:pce4-configuration.adoc[Configure PCE 4 in Studio]
- ** xref:pce3-configuration.adoc[Configure PCE 3 in Studio]
- *** xref:pce-configuration-macos.adoc[MacOS]
- *** xref:pce-configuration-windows.adoc[Windows]
-
 * xref:login-platform-from-studio.adoc[]
  ** xref:set-credentials-in-studio-to.adoc[]
 
+* xref:set-anypoint-platform-cloud.adoc[Set Anypoint Platform Cloud]
+ ** xref:deploy-mule-application-task.adoc[Deploy a Mule Application to CloudHub]
```

Comment:
In general, we use gerunds (and plural) for H1s if there is more than one task in the file:
Deploying Mule Apps to CloudHub

Reply:
OK! Thanks!